.github/workflows/stale.yml

```diff
@@ -27,18 +27,18 @@ jobs:
         with:
           repo-token: '${{ secrets.GITHUB_TOKEN }}'
           stale-issue-message: >-
-            This issue has been automatically marked as stale due to 60 days of inactivity.
+            This issue has been automatically marked as stale due to 14 days of inactivity.
             It will be closed in 14 days if no further activity occurs.
           stale-pr-message: >-
-            This pull request has been automatically marked as stale due to 60 days of inactivity.
+            This pull request has been automatically marked as stale due to 14 days of inactivity.
             It will be closed in 14 days if no further activity occurs.
           close-issue-message: >-
             This issue has been closed due to 14 additional days of inactivity after being marked as stale.
             If you believe this is still relevant, feel free to comment or reopen the issue. Thank you!
           close-pr-message: >-
             This pull request has been closed due to 14 additional days of inactivity after being marked as stale.
             If this is still relevant, you are welcome to reopen or leave a comment. Thanks for contributing!
-          days-before-stale: 60
+          days-before-stale: 14
           days-before-close: 14
```

Comment:
should we reduce days before close too? maybe 7?

that way, end to end, it takes 3 weeks of inactivity?